\chapter*{Abstract}
\chaptermark{abstract}
\addcontentsline{toc}{chapter}{Abstract}

As a result of the process of descent with modification, closely related species share many traits. Phylogeny thus provides information that not only needs to be considered when making inter-specific comparisons but which also can be leveraged to gain insight into macroevolutionary questions. Here I develop statistical methods, computational tools, and conceptual frameworks to help researchers make more meaningful inferences from phylogenetic comparative data. 

In my opening chapter, I provide a theoretical foundation of how researchers can use models of trait evolution to test hypotheses related to the long-controversial theory of punctuated equilibrium (PE), which asserts that speciation causes rapid evolution against a backdrop of stasis. I break the hypothesis down into four key elements and argue that combining these conceptually distinct ideas under the single framework of PE is distracting and confusing, and more likely to hinder progress than to spur it. 

Next, I present a suite of statistical software, written in the R programming language, for fitting evolutionary models to phylogenetic data. This is a complete overhaul of the popular \textsc{geiger} package designed to facilitate analyses of large and complex comparative datasets.

I then use phylogenetic models of discrete character evolution to estimate the relative rates of autosomal fusions for the four different types of sex chromosomes (X, Y, Z, and W) in fish and squamates reptiles. I find that Y-autosome fusions occur much more frequently than any other type of sex chromosome-autosome fusion. This result grounded a theoretical investigation into the evolutionary forces driving sex chromosome fusions--the phylogenetic results allowed my collaborators and I to exclude from consideration several existing theories for why fusions become fixed in populations. In addition to providing novel insights into chromosomal evolution, this study demonstrates the potential for phylogenetic models to complement more conventional population-genetic modeling.

In the final two chapters, I address two outstanding statistical problems that hinder the use and interpretation of phylogenetic models of trait evolution. First, I develop a novel statistical framework for assessing the absolute fit, or adequacy, of phylogenetic models of trait evolution. To date, most researchers have focused almost exclusively on the relative explanatory power of alternative models, rather than the ability of a model to provide a good explanation for data on its own terms. I use my approach to evaluate the statistical performance of commonly used trait models on 337 comparative datasets covering three key Angiosperm (``flowering plants'') functional traits. In general, the models we tested often provided poor statistical explanations for the evolution of these traits. This was true for many different groups and at many different scales. Second, I develop a new technique that uses taxonomic information to assess and overcome sampling biases in trait datasets. I use this to provide the first estimate of the global distribution of woody and herbaceous plants from a database of 39,313 records and find that the world is likely much woodier than researchers previously thought. It is an exciting time for macroevolutionary research as the scale of data has grown tremendously in the last few years. The work presented here will hopefully help researchers make the most of it.

%%% Local Variables:
%%% TeX-master: "thesis"
%%% TeX-PDF-mode: t
%%% End:
